Custom/annatar.yml

```diff
@@ -6,69 +6,58 @@ type: public
 encoding: UTF-8
 followredirect: false
 testlinktorrent: false
-requestDelay: 15
+requestDelay: 2
 links:
   - https://annatar.elfhosted.com
   - http://annatar:8000
 caps:
   categories:
     Movies: Movies
     TV: TV
-
   modes:
     search: [q]
-    movie-search: [q, imdbid]
-    tv-search: [q, imdbid, season, ep]
+    movie-search: [imdbid]
+    tv-search: [imdbid]
```

Comment:
these 2 are wrong, `q` is actually required

Reply:
also, dont remove the ability to use `season` and `ep` as well. Different indexers use these that support them. Typically, if they support imdbid then they also support season and ep params

Reply:
> these 2 are wrong, `q` is actually required

Only required for the search parameter.

Reply:
> also, dont remove the ability to use `season` and `ep` as well. Different indexers use these that support them. Typically, if they support imdbid then they also support season and ep params

Guess it won't hurt keeping them.

Reply:
> Only required for the search parameter.

Thats strange, when I was working on the Torrentio indexer.. it was yelling at me for trying to remove the `q` and wanted all the modes to have it by default 🤔.. strange.

If it works without, then awesome!